specs/lib/removePipelineExtrasSpec.js

```diff
@@ -7,7 +7,7 @@ describe('removePipelineExtras', function() {
     var gltf;
 
     beforeAll(function(done) {
-        fs.readFile(gltfExtrasPath, function(err, data) {
+        fs.readFile(gltfExtrasPath, function(err, data) { //eslint-disable-line no-unused-vars

```

Comment:
Remove this comment and fix the function by adding

```javascript
if(defined(err) {
    done.fail(err);
    return;
}
```
to the top of the function.